eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -138,9 +150,12 @@ class SqliteAuditDb(val sqlite: Connection) extends AuditDb with Logging {
         statement.executeUpdate("CREATE INDEX metrics_timestamp_idx ON path_finding_metrics(timestamp)")
         statement.executeUpdate("CREATE INDEX metrics_mpp_idx ON path_finding_metrics(is_mpp)")
         statement.executeUpdate("CREATE INDEX metrics_name_idx ON path_finding_metrics(experiment_name)")
+        statement.executeUpdate("CREATE INDEX metrics_recipient_idx ON path_finding_metrics(recipient_node_id)")
+        statement.executeUpdate("CREATE INDEX metrics_hash_idx ON path_finding_metrics(payment_hash)")
+        statement.executeUpdate("CREATE INDEX metrics_hint_idx ON path_finding_metrics(routing_hint_node_ids)")
```

Comment:
Does that really make sense? Since this is a `TEXT` field, does the index let you efficiently filter on a substring? I imagine that if we want to filter on routing hints, we would filter on only one routing hint `node_id` but want to find all payments for which it was one of the routing hints?

Reply:
I'm assuming that this field will contain 0 or 1 node id

Reply:
It's probably fine, but I'm just worried about users with sqlite on the same machine as their node, I'd like to avoid impacting their node performance as much as possible. Pierre-Marie probably has more experience with that, let's see what he thinks.

Reply:
I would not bother implementing on sqlite. It's advanced stuff and we need json support for proper querying anyway.

Reply:
Re: index and performance, why not just add the index later if it turns it's needed?